src/anyp/UriScheme.cc

```diff
@@ -48,6 +48,25 @@ AnyP::UriScheme::Init()
     }
 }
 
+const AnyP::ProtocolType
+AnyP::UriScheme::FindProtocolType(const SBuf &str)
+{
+    if (str.isEmpty())
+        return AnyP::PROTO_NONE;
```

Comment:
If possible, please avoid `AnyP::` inside AnyP.

Reply:
PROTO is a system macro. Some of this Enum's values collide.

Reply:
On Windows the PROTO gets macro replaced with a (...) code production.

Reply:
I do not understand because I see no `PROTO` names in this context. Are you implying that, on Windows, some compiler takes the name `PROTO_NONE` and replaces the first five letters with something, as if those letters were a stand-alone name?!

Reply:
Something in the build chain does that yes. I doubt it is the compiler directly though.

Reply:
If something replaces letters `PROTO` with a `(...) code production` (whatever that means), then how can Squid code compile with or without your changes? Squid already has code that has bare PROTO_NONE and similar names. Is that "something in the build chain" so tricky that it replaces some names but not others?

Anyway, this is not important enough to drag on. If you are sure it is not possible to remove `AnyP:` prefix, let's keep it.